source/_components/cover.mqtt.markdown

```diff
@@ -159,6 +159,35 @@ unique_id:
   description: An ID that uniquely identifies this cover. If two covers have the same unique ID, Home Assistant will raise an exception.
   required: false
   type: string
+device:
+  description: 'Information about the device this cover is a part of to tie it into the [device registry](https://developers.home-assistant.io/docs/en/device_registry_index.html). Only works through [MQTT discovery](/docs/mqtt/discovery/) and when [`unique_id`](#unique_id) is set.'
```

Comment:
`Information about the device this cover` sounds weird.

Reply:
Ok, I don't really know how to put it else. I think one option would be to just remove the second part of that: `Information about the device to tie it into the device registry.`

Reply:
I've been thinking about another wording...
I was just saying it sounds weird / doesn't flow. But it is correct.
So let's leave it like it is.